connector/plugin/plugin.go

```diff
@@ -0,0 +1,46 @@
+package plugin
+
+import (
+	plug "plugin"
+
+	"github.com/dexidp/dex/connector"
+	"github.com/sirupsen/logrus"
```

Comment:
[nit] Could we separate `github.com/dexidp/dex/...` imports into a third group, please? Like `goimports -local github.com/dexidp/dex` would do it.